### ISSUE TITLE
Remove problematic details tag from doc-site.

### ISSUE THREAD

**Issue by pranav300** [cerner/terra-core#2649]
# Bug Report

## Description

The inert polyfill doesn't make `details` tag go out of `tab order`, which causes an accessibility violation in the doc-site.
